src/docfx/build/metadata/SystemMetadata.cs

```diff
@@ -31,6 +31,9 @@ internal class SystemMetadata
         [JsonProperty("_op_gitContributorInformation")]
         public ContributionInfo? ContributionInfo { get; set; }
 
+        [JsonProperty("git_contributor_information")]
```

Comment:
>   [JsonProperty("git_contributor_information")] [](start = 5, length = 48)

remove. it's already snake case naming convention

Reply:
Oh...Sorry...